bot/music_player/player_cog.py

```diff
@@ -0,0 +1,121 @@
+from discord import voice_client
+from discord.ext import commands
+from discord.player import FFmpegAudio
+import ffmpeg
+
+#import bot.music_player.player
+from bot.music_player.player import Player
+
+# Manages all of the players. 
+# Handles requests directly from discord, passing them along to their appropriate players
+class PlayerCog(commands.Cog):
+
+    def __init__(self):
+        self.players = {}
+
+    def create_player_if_needed(self, guild_id):
+        if not guild_id in self.players.keys():
+            self.players[guild_id] = Player(guild_id)
+
+    @commands.command(
+        brief="Adds bot to voice channel",
+        help="""Commands bot to join the voice channel the commanding user is in""",
+        name="join_voice"
+    )
+    async def join_voice(self, ctx):
+        if not ctx.message.author.voice:
+            await ctx.send("{} is not connected to a voice channel".format(ctx.message.author.name))
+            return
+        elif ctx.voice_client and ctx.voice_client.is_connected():
+            await ctx.send("Bot is already connected to a voice channel.")
+            return
+        else:
+            self.create_player_if_needed(ctx.message.guild.id)
+            channel = ctx.message.author.voice.channel
+        await channel.connect()
+
+
+    @commands.command(
+        brief="Removes bot from voice channel",
+        help="""Removes the bot from whichever voice channel it is currently in.""",
+        name="leave_voice"
+    )
+    async def leave_voice(self, ctx):
+        await ctx.voice_client.disconnect()
+        del self.players[ctx.message.guild.id]
+
+
+    @commands.command(
+        brief="Plays a single song",
+        help="""Plays a single song given an mp3's name. If a song is already playing,
+        it adds the song to a queue to be played when the current song ends. Will
+        automatically add bot to a voice channel if its not in one already.""",
+        name="play_local"
+    )
+    async def play_local(self, ctx, song_name, filter=""):
```

Comment:
If you want to use multiword song names without quotation marks, you can remove the filter (perhaps make the filter a command per-guild?) argument and use the following function definition instead.
```suggestion
    async def play_local(self, ctx, *, song_name):
```

Reply:
After discussion, this sort of change would be in a separate PR.  We'll leave that for later and accept as-is.